pkg/mount/mount_linux.go

```diff
@@ -74,10 +74,17 @@ func (mp *MountPoint) Unmount(flags uintptr) error {
 // dev is the device to mount (this is often the path of a block device, name
 // of a file, or a placeholder string). data usually contains arguments for the
 // specific file system.
-func Mount(dev, path, fsType, data string, flags uintptr) (*MountPoint, error) {
+//
+// opts is usually empty, but if you want, e.g., to pre-create the mountpoint,
+// you can call Mount with a mkdirall, e.g.
+// mount.Mount("none", dst, fstype, "", 0,
+//		func() error { return os.Mount(dst, 0o666)})
+func Mount(dev, path, fsType, data string, flags uintptr, opts ...func() error) (*MountPoint, error) {
 	// Create the mount point if it doesn't already exist.
```

Comment:
Isn't this comment too specific? Looping `opts` could do anything.

Reply:
oops! I left it in by mistake.